src/components/PopoverWithMeasuredContent.js

```diff
@@ -30,6 +32,13 @@ const propTypes = {
     but in the case the children are not displayed, the measurement will not work. */
     measureContent: PropTypes.func.isRequired,
 
+    /** Static dimensions if the popover height, width is known beforehand.
+     * When passed, it will skip measuring popover on render */
```

Comment:
```suggestion
    /** 
     * Static dimensions for the popover.
     * Note: When passed, it will skip dimensions measuring of the popover, and provided dimensions will be used to calculate thenchor position.
     */
```

Please fix the linting issues in the suggestions.